sys/net/gnrc/doc.txt

```diff
@@ -0,0 +1,163 @@
+/**
+ * @defgroup    net_gnrc Generic (gnrc) network stack.
+ * @ingroup     net
+ * @brief       RIOT's modular default IP network stack.
+ *
+ * # About
```

Comment:
Nitpicky, but I prefer the

```markdown
Section
======

Subsection
----------
```

syntax over the 

```markdown
# Section

## Subsection
```

syntax. I always think those are comments ^^